indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -479,4 +497,7 @@ <h3 class="abstract-title">
             </a>
         {% endif %}
     </div>
+    <script>
+        $('.dropdown-abstract-pdf').parent().dropdown();
```

Comment:
Yes, we have `.js-dropdown` but it doesn't work when there are AJAX reloads in the page as I mention in a comment before. If you know how to solve it please let me know.